test/ping-pong/stop-ping_pong.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
```

Comment:
Do we need this file?

Reply:
No, I will delete these files because now I have the solution with docker-compose directly.